.github/workflows/run-build-and-acceptance-tests.yml

```diff
@@ -119,7 +120,11 @@ jobs:
         with:
           dotnet-version: ${{ matrix.dotnet-version }}
       - run: mkdir -p ${{ runner.temp }}/opt/pulumi/nuget
-      - run: dotnet nuget add source ${{ runner.temp }}/opt/pulumi/nuget
+      - name: Add NuGet packages as a local NuGet source
```

Comment:
This is needed in lint job?

Reply:
Not sure, didn't want to make a bigger change than necessary, just wanted to log the version for posterity.